checks/evaluation/code_review.go

```diff
@@ -57,6 +58,13 @@ func CodeReview(name string, dl checker.DetailLogger, r *checker.CodeReviewData)
 }
 
 func reviewScoreForChangeset(changeset *checker.Changeset) (score reviewScore) {
+	if changeset.Commits[0].Committer.IsBot {
```

Comment:
is it right to always ignore these bots? Say, AllStar / Dependabot sends a PR to fix something.. should it be manually reviewed? Ignoring unilaterally creates a precedent that any random bots someone sets up won't need reviews.
Is there a way to distinguish?

Also, I think it's feasible for someone to set the bot name themselves when creating a PR / commit. 

Shall we use the User.id instead https://github.com/raghavkaul/scorecard/blob/5b490d3e583bbb1b774a75c459caaf0496292a90/clients/user.go#L23 to identify "real" bots? I think this one cannot be forged... but should verify my claim.

If this works, shall we keep a list of "acceptable" bots? And add it to our documentation of the check?